# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: package/.../e2fsprogs/install.patch
# Copyright (C) 2007 The OpenSDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

We can't have absolute symlinks because on stage 1 they mix with the host -amery

--- ./lib/Makefile.elf-lib.orig	2007-10-12 10:16:52.000000000 -0400
+++ ./lib/Makefile.elf-lib	2007-10-12 10:17:47.000000000 -0400
@@ -45,7 +45,7 @@
 	@echo "	SYMLINK $(ELF_INSTALL_DIR)/$(ELF_SONAME)"
 	@$(LN_S) -f $(ELF_LIB) $(DESTDIR)$(ELF_INSTALL_DIR)/$(ELF_SONAME)
 	@echo "	SYMLINK $(libdir)/$(ELF_IMAGE).so"
-	@$(LN_S) -f $(ELF_INSTALL_DIR)/$(ELF_SONAME) \
+	@$(LN_S) -f $(ELF_SONAME) \
 		$(DESTDIR)$(libdir)/$(ELF_IMAGE).so
 	@echo "	LDCONFIG"
 	@-$(LDCONFIG)
